.github/workflows/ci-dgraph-tests.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           go mod tidy
           make regenerate
           git diff --exit-code -- .
+      - name: Make Docker Image
+        run: make image-local
```

Comment:
One thing I'm working about: for the Github Action that's not a custom runner, are we sure there's a local docker registry that this is getting tagged in? I guess, the answer is "yes" because dgraph/dgraph:local is never pushed to a remote image registry.

Reply:
Yes is the answer. 

I validated this on the `runner` to make sure this gets referenced correctly. And it does! I also went one step above, I cleaned up all the `stale` docker images pointing to `latest` on the runner. And since the merge, the machine has not pulled in `dgraph/dgraph:latest`